rero/rero-ils
# Circulation: Circ policies backend

NEW: Integrate Circ policies with invenio circulation

Signed-off-by: Aly Badr <aly.badr@rero.ch>

How to test:
- ./scripts/bootstrap
- ./scripts/setup
- localhost:5000, place checkout/renewal and make sure due date is correctly calculated according to configured circ policies
